src/test/java/column/store/api/column/ColumnTest.java

```diff
@@ -19,7 +18,7 @@ class ColumnTest {
 
     @ParameterizedTest
     @ValueSource(strings = { "someColumn", "YET_ANOTHER_COLUMN", "aVeRyWeIrDcOlUmN"})
-    void columnNamesAreLowercase(String name) {
+    void columnNamesAreLowercase(final String name) {
         var booleanColumn = Column.forBoolean(name);
```

Comment:
Should local variables also be `final` if possible?

Reply:
would be possible, but don't really think this is a good idea/necessary